tests/selectedtests/conftest.py

```diff
@@ -211,6 +211,39 @@ def _repo(temp_directory):
     return _repo
 
 
+@pytest.fixture(scope="function")
+def repo_with_interval_commits(monkeypatch):
```

Comment:
[nit] This is test code, so I tend to be more flexible, but this fixture is pretty complicated. If you think it would be useful to be used in other tests, I think a comment describing what it does would be useful.

Reply:
👍 removed